### PR TITLE
geoseries ep 16 close #59

### LINKED ISSUE #59: add geoseries ep 16 details in the github.io page

### PR DESCRIPTION
**Pull request by iamtekson**
## What does this PR do? 

- geoseries ep 16

### CODE REVIEW

File: docs/Events/2024_06_21_geoseries_16.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+**Poster By:** Aayush Chand
+
+**Presentor:** Pratichhya Sharma
+
+**Affiliation:** Remote Sensing Data Engineer at VITO
+
+### Short Description
+
+Pratichhya is a Remote Sensing Data Engineer at VITO which is the Flemmish Institute for Technological Research. Her research work focuses on efficient remote sensing data analysis using cloud based solutions such as openEO. Currently she is actively involved in Copernicus Dataspace Ecosystem project where the European Space Agency aims to centralize data and APIs. Within the project she also tries to highlight the importance of FAIR principles and open-science offered by different components of ecosystem. Additionally she has also been an active member of the OSGeo Nepal community with an aim to encourage them in using open-source tools and technologies.
```
Comment (Pratichhya) [inline docs/Events/2024_06_21_geoseries_16.md:17]:
```suggestion
Pratichhya is a Remote Sensing Data Engineer at VITO (Flemish Institute for Technological Research). Her research focuses on efficient remote sensing data analysis using cloud-based solutions such as openEO. Currently, she is involved in the Copernicus Dataspace Ecosystem project, where the European Space Agency aims to centralize data and APIs. Within the project, she also tries to highlight the importance of FAIR principles and open science offered by different components of the ecosystem. Additionally she has also been an active member of the OSGeo Nepal community to encourage them in using open-source tools and technologies.
```

**Review by Pratichhya — APPROVED** [review]

**Comment by Pratichhya** [comment]
Except for these slight changes, others look good to me.

**Comment by iamtekson** [comment]
@Pratichhya dd, can you please share the slides link if possible?

**Comment by Pratichhya** [comment]
Link additional materials used in the presentation:
https://youtu.be/jCg4u8gPxlA
https://github.com/Open-EO/openeo-community-examples/tree/main/python

For slides I will coordinate with you @iamtekson

**Comment by iamtekson** [comment]
I am still waiting for the slides @Pratichhya dd!

**Comment by Pratichhya** [comment]
https://docs.google.com/presentation/d/1Q841DcNCprNMH1di7Brz4r-X9ms4D1M-/edit?usp=drive_link&ouid=114236913747112763251&rtpof=true&sd=true

**Comment by iamtekson** [comment]
great! thank you dd!

**Pull request merged**